chxtools/speckle.py

```diff
@@ -10,8 +10,8 @@
 import numpy as np
 import time
 
-from skxray.core import roi
-from skxray.core.utils import bin_edges_to_centers, geometric_series
+from skbeam.core import roi
```

Comment:
for some history: `scikit-xray` was renamed to `scikit-beam` a while ago. So just substituting the name is perfect.